engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/EvalTest.scala

```diff
@@ -36,6 +36,20 @@ class EvalTest extends InterpreterTest {
       consumeOut shouldEqual List("Hello World!")
     }
 
+    "have access to the caller scope with <|" in {
+      val code =
+        s"""import Standard.Base.Runtime.Debug
+           |import Standard.Base.IO
+           |
+           |main =
+           |    x = "Hello World!"
+           |    Debug.eval <| $rawTQ
```

Comment:
The combination of `Debug.eval` and `<|` was always broken. The `<|` builtin was providing its own `VirtualFrame` to the `Debug` code. Now the `<|` builtin specifies `needsFrame = false` and gets the caller's frame instance. As such it provides the right frame to the `Debug` subsystem.